draft-ietf-rats-corim.md

```diff
@@ -550,31 +550,33 @@ A CoRIM role entry that contains the `manifest-signer` role MUST be added to `co
 
 It is out of scope of this document to specify a method of delegating the signer role in the case that an unsigned CoRIM is conveyed through multiple secured links with different notions of authenticity without end-to-end integrity protection.
 
-### CoRIM bundles
+### CoRIM collections
 
-A method of signing a bundle of CoRIMs together is through a signed RATS Conceptual Message Wrapper (CMW) {{-cmw}}.
-The COSE_Sign1 signature format can be used with a Collection CMW.
-The COSE protected header can include a Collection CMW type name.
-The collection type name SHALL be of the form: `tag:{{&SELF}}:bundle`.
+Several CoRIMs may share the same signer and use locally-resolvable references to each other by using a signed RATS Conceptual Message Wrapper (CMW) {{-cmw}}.
```

Comment:
Can a Collection CMW be an unsigned payload within a secure session or by some other enveloping technology such as PKIX? If yes, then this language may be too constraining. Does "signed RATS CMW" means COSE_Sign1 or could it be interpreted more broadly? For example, it could read: "Several CoRIMs may share the same signer (e.g., as a payload to a secure session protocol or signature format such as COSE) and use locally-resolvable references to each other by using a RATS Conceptual Message Wrapper (CMW) {{-cmw}}." Note reference to COSE is exemplary.